# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/libm.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Well - this is just to get my ARM build going - we should fill them soon ;-)

  - Rene Rebe <rene@exactcode.de>

--- dietlibc-0.28/libm/__halfc.c	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.28-fixed/libm/__halfc.c	2005-02-03 18:52:53.000000000 +0100
@@ -0,0 +1,6 @@
+
+#if !(defined (__i386__))
+
+float __half = 0.5;
+
+#endif
--- dietlibc-0.28/libm/cosc.c	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.28-fixed/libm/cosc.c	2005-02-03 18:52:27.000000000 +0100
@@ -0,0 +1,12 @@
+#include <math.h>
+#include <stdlib.h>
+
+#if !(defined (__i386__))
+
+double  cos ( double x )
+{
+    __write2("No sin yet.");
+    exit (1);
+}
+
+#endif
--- dietlibc-0.28/libm/expc.c	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.28-fixed/libm/expc.c	2005-02-03 18:55:51.000000000 +0100
@@ -0,0 +1,12 @@
+#include <math.h>
+#include <stdlib.h>
+
+#if !(defined (__i386__))
+
+double  exp ( double x )
+{
+    __write2("No exp yet.");
+    exit (1);
+}
+
+#endif
--- dietlibc-0.28/libm/floorc.c	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.28-fixed/libm/floorc.c	2005-02-03 18:54:54.000000000 +0100
@@ -0,0 +1,12 @@
+#include <math.h>
+#include <stdlib.h>
+
+#if !(defined (__i386__))
+
+double  floor ( double x )
+{
+    __write2("No floor yet.");
+    exit (1);
+}
+
+#endif
--- dietlibc-0.28/libm/fmodc.c	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.28-fixed/libm/fmodc.c	2005-02-03 18:53:32.000000000 +0100
@@ -0,0 +1,13 @@
+#include <math.h>
+#include <stdlib.h>
+
+#if !(defined (__i386__))
+
+double  fmod ( double x, double y )
+{
+    __write2("No fmod yet.");
+    exit (1);
+}
+
+#endif
+
--- dietlibc-0.28/libm/logc.c	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.28-fixed/libm/logc.c	2005-02-03 18:55:17.000000000 +0100
@@ -0,0 +1,12 @@
+#include <math.h>
+#include <stdlib.h>
+
+#if !(defined (__i386__))
+
+double  log ( double x )
+{
+    __write2("No log yet.");
+    exit (1);
+}
+
+#endif
--- dietlibc-0.28/libm/sinc.c	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.28-fixed/libm/sinc.c	2005-02-03 18:52:18.000000000 +0100
@@ -0,0 +1,12 @@
+#include <math.h>
+#include <stdlib.h>
+
+#if !(defined (__i386__))
+
+double  sin ( double x )
+{
+    __write2("No sin yet.");
+    exit (1);
+}
+
+#endif
--- dietlibc-0.28/libm/sqrtc.c	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.28-fixed/libm/sqrtc.c	2005-02-03 18:55:28.000000000 +0100
@@ -0,0 +1,12 @@
+#include <math.h>
+#include <stdlib.h>
+
+#if !(defined (__i386__))
+
+double  sqrt ( double x )
+{
+    __write2("No sqrt yet.");
+    exit (1);
+}
+
+#endif
--- dietlibc-0.28/libm/tanc.c	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.28-fixed/libm/tanc.c	2005-02-03 18:52:35.000000000 +0100
@@ -0,0 +1,12 @@
+#include <math.h>
+#include <stdlib.h>
+
+#if !(defined (__i386__))
+
+double  tan ( double x )
+{
+    __write2("No tan yet.");
+    exit (1);
+}
+
+#endif
